src/control/cmd/dmg/pretty/storage.go

```diff
@@ -176,15 +176,28 @@ func PrintStorageFormatMap(hsm control.HostStorageMap, out io.Writer, opts ...Pr
 }
 
 func printSmdDevice(dev *storage.SmdDevice, iw io.Writer, opts ...PrintConfigOption) error {
-	if _, err := fmt.Fprintf(iw, "UUID:%s [TrAddr:%s]\n",
-		dev.UUID, dev.TrAddr); err != nil {
+	fc := getPrintConfig(opts...)
 
-		return err
+	if fc.LEDInfoOnly {
```

Comment:
I'm following the current model of the SMD query commands which end up calling `cmd.makeRequest()` which in turn prints out the SmdInfo. Supplying the PrintConfigOption enables us to continue using this concise form otherwise there would be some duplication creating a near identical `makeRequest` function just for the two LED commands.